gw_eccentricity/eccDefinition.py

```diff
@@ -303,6 +323,31 @@ def __init__(self, dataDict, num_orbits_to_exclude_before_merger=2,
                 eccentricity is the cause, and set the returned eccentricity
                 and mean anomaly to zero.
                 USE THIS WITH CAUTION!
+
+            omega_gw_extrema_interpolation_method : str, default="rational_fit"
```

Comment:
Move this above, to before special_interp_kwargs_for_extrema?

Reply:
Done